jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -37,6 +37,23 @@ YUI.add('juju-env-api', function(Y) {
   var utils = Y.namespace('juju.views.utils');
 
   /**
+    Return a proper local charm endpoint for both GIJoe and charm
```

Comment:
GIJoe is not an official name, GUI in Juju sounds worse but it would be more correct here. [shrug]